packages/selleo-design-core/src/parser.tsx

```diff
@@ -17,9 +17,11 @@ export const parseComponent = (html: string) => {
   const Component = buildComponent(html);
   const HTMLString = render(<Component />, null, { pretty: true });
   const ReactString = parseToReact(HTMLString);
+  const HTMLStringResult = HTMLString.replaceAll(/><\/input>|><\/img>/g, '>');
+  const ReactStringResult = ReactString.replaceAll(/><\/input>|><\/img>/g, '>');
```

Comment:
That's the same as HTML, with React we would like to have the proper react self closing elements `<element />` not `<element>`

Reply:
@DawidHaratyk please add this small change and it will be approved :>